projects/saturn/src/test/java/io/fairspace/saturn/vocabulary/VocabulariesTest.java

```diff
@@ -27,11 +30,15 @@ public void setUp() {
 
     @Test
     public void validateVocabulary() {
-        validate(VOCABULARY, SHACL_FOR_SHACL);
+        var dsg = DatasetGraphFactory.createTxnMem();
+        Dataset ds = wrap(dsg);
+        Model model = ds.getDefaultModel();
+        var vocabulary = model.read("test-vocabulary.ttl");
```

Comment:
is it possible to use the loadModel() function instead? 

```suggestion
    private static final Model TEST_VOCABULARY = loadModel("test-vocabulary.ttl");
```